packages/circuits/helpers/remove-soft-line-breaks.circom

```diff
@@ -0,0 +1,115 @@
+pragma circom 2.1.6;
+
+include "circomlib/circuits/comparators.circom";
+include "circomlib/circuits/mux1.circom";
+include "../utils/hash.circom";
+
+template RemoveSoftLineBreaks(maxLength) {
+    signal input encoded[maxLength];
+    signal input decoded[maxLength];
+    signal output is_valid;
```

Comment:
We have been using camelCase for signal variables names. Would be nice to be consistent at-least for input/output signals.

Reply:
Thanks for pointing it out. Made the changes :)